### ISSUE TITLE
 DS iPlayer ( www.dsiplayer.com)

### ISSUE THREAD

**Issue by Dosseum** [kitling/flashcart_core#45]
The last firmware : http://soft.dsiplayer.com/eng/eng_iPlayer_OS_20120323.zip (it was for dsi 1.4.4 but worked on 3DS at beggining if i remember well, and doesn't work anymore (it show the game but blank when i launch) ) (it contains update.dat to update the firmware)

There you can have some other download (the old firmware, DLDI, etc..) : http://www.dsiplayer.com/download.html

It was make by the supercard team, and it has "some chip" to do emulator GBA working like the DSTwo. It can only run homebrew .nds  (for the NDS thing), because the purpose was to be a video player, but it was made like a flashcard and can be updated and launch .nds hombrew (and gba rom)

The "nds game" used is Fish Tycoon

ChipID: F9FEB7FE
HW Revision: F9FEB7FE

<details><summary>Images <em>(Click to Expand)</em></summary><p> 

![20170903_162149](https://user-images.githubusercontent.com/25887041/30003895-861c2d00-90c5-11e7-9243-ccf64d8d15b7.jpg)
![20170903_162303](https://user-images.githubusercontent.com/25887041/30003896-861ebf48-90c5-11e7-978e-82f4de36c28b.jpg)
![20170903_161530](https://user-images.githubusercontent.com/25887041/30003897-8621887c-90c5-11e7-9ce7-64ffd40aca5c.jpg)
![20170903_161546](https://user-images.githubusercontent.com/25887041/30003898-86249986-90c5-11e7-81a0-fde970ff987b.jpg)

</p></details> 

Number on chips (maybe not exact) : 1) AML 3405 VQ100,0905 CC180Z9801
2) SST 39VF040 70-4C-WHE 0852077-L
3) wynix 537A HY5?V581620GLT 
4) TMS 320 DM 6541 AZWG

GM9 Untrimmed Dump: 
FISHTYCOON_YF7E5G_00.nds : http://www46.zippyshare.com/v/U3UQCa5n/file.html

**Comment by ShadixAced** [comment]
Wow, pretty decent issue post you have there. With all the elements you have there, it will be easy to support that card.

**Comment by dark-samus** [comment]
no, it will not be "easy" to support this card. This card is made by supercard, and is a lot like the SCDS2. However, it's much more rare. The update is encrypted it would seem, so I can't just take a look at it easily, and without an actual cart, it might be hard to reverse the crypto. It seems to differ from SCDS2 crypto too... tl;dr, unless I can figure out how to decrypt this thing, support will be hard to add.

**Comment by ShadixAced** [comment]
Was talking about the fact that it will be easy to understand the cart since he put everything related

**Comment by dark-samus** [comment]
what are you even saying

**Comment by Dosseum** [comment]
Thank for the reply ! Well i tried to send a mail to dsiplayer.com about their update file in hope to have a reply... hahaha But yes it's more rare :/ So maybe it's not the most important flashcart to do (i've a gateway blue & action replay & ds one haha)
